### DISCUSSION TITLE
DKIM not verifying for a domain

### DISCUSSION THREAD

**Discussion by sheikhsherazali** [postalserver/postal#1813] [Help with using Postal]
Dear Support Team 

First of all many thanks for your wonderful support. 

The issue I am facing is that my DKIM is not validating 

![image](https://user-images.githubusercontent.com/13349939/149674942-33d3de70-14f9-4ec2-bd0d-c0536f42c404.png)

Although I have the exact DKIM record at my name servers. Please see the screenshot  

![image](https://user-images.githubusercontent.com/13349939/149675042-f9d80e93-3951-4a9e-9903-17dbb3acd331.png)

Please can you help me out with what is causing this?

**Comment by willpower232** [comment]
You should use the terminal on your server to try the `dig` command to verify that your server can see the record. If it can't you can also try whatsmydns.net to make sure the record works worldwide.

**Reply by sheikhsherazali** [reply]
Hi, It's working fine, and many thanks for such wonderful support. 

I need to update the records at the Cloudflare side this is causing the error. Its working perfectly fine now

Please can you close this ticket thanks?

**Reply by sheikhsherazali** [reply]
I am closing the ticket as well. 

Thanks.

**Reply by willpower232** [reply]
glad you figured it out
